app/bench_test.go

```diff
@@ -131,11 +174,25 @@ func benchmarkERC20Transfer(b *testing.B, db dbm.DB) {
 	var transferTxs [][]byte
 	for i := 0; i < b.N; i++ {
 		for j := 0; j < txsPerBlock; j++ {
-			idx := int64(i*txsPerBlock + j)
-			recipient := common.BigToAddress(big.NewInt(idx))
+			idx := rand.Int() % len(testAccounts)
+			acct := &testAccounts[idx]
+			recipient := common.BigToAddress(big.NewInt(int64(idx)))
 			data, err := types.ModuleCRC21Contract.ABI.Pack("transfer", recipient, big.NewInt(1))
 			require.NoError(b, err)
-			bz, err := signTx(evmtypes.NewTx(chainID, uint64(idx), &contractAddr, big.NewInt(0), 210000, gasPrice, nil, nil, data, nil))
+			tx := evmtypes.NewTx(
+				chainID,
+				acct.Nonce,    // nonce
+				&contractAddr, // to
+				big.NewInt(0), // value
+				210000,        // gas limit
+				nil,           // gas price
+				gasPrice,      // gasFeeCap
```

Comment:
```suggestion
				gasPrice,      // gas price
				nil,           // gasFeeCap
```

Reply:
I try to test dynamic fee here, it's more representative I think